packages/babel-make-styles/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui/babel-make-styles",
+  "version": "9.0.0-alpha.0",
+  "description": "A Babel plugin that performs build time transforms for @fluentui/make-styles",
+  "main": "lib/src/index.js",
+  "typings": "lib/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.1.0",
+    "@fluentui/react-make-styles": "^9.0.0-alpha.23",
+    "@fluentui/scripts": "^1.0.0",
+    "babel-plugin-tester": "^10.0.0",
+    "typescript": "4.2.4"
+  },
+  "dependencies": {
+    "@babel/core": "^7.10.4",
+    "@babel/generator": "^7.12.13",
+    "@babel/helper-plugin-utils": "^7.12.13",
+    "@babel/template": "^7.12.13",
+    "@babel/traverse": "^7.12.13",
```

Comment:
Should this be peer deps ? When you setup a project, you'd already have babel deps generally

Reply:
Linaria declares them as `dependencies`:
https://github.com/callstack/linaria/blob/3c56c067edf4d1a3563b121e476b3400e5e5e720/packages/babel/package.json#L48-L50

I don't know what will be a better option in this case. If we will have a missing dependency, it will fail to run at all 🤨

Reply:
> If we will have a missing dependency, it will fail to run at all

Since peer deps are generally considered to be required, I think this is okay? But I don't have a strong opinion on deps vs. peer deps here.

Reply:
I will go with deps for now, but I am ready to change it once we will try to integrate it into products.